.github/workflows/periodic-integration-tests.yml

```diff
@@ -0,0 +1,90 @@
+name: Periodic Integration Tests
+
+on:
+  schedule:
+     - cron: '0 15 * * 1-5' # Run at 7AM PST on every weekday
+  workflow_dispatch:
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        # These are all the Python versions that we support.
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        concurrency: [1, 2, 5]
+
+    name: Run Integration Tests with Python Version ${{ matrix.python-version }} Concurrency ${{ matrix.concurrency }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Create the logs directory
+        run: mkdir -p logs
+
+      - name: Set filename for the server's output logs
+        run: echo "SERVER_LOGS_FILE=logs/server_logs_${{ matrix.python-version }}" >> $GITHUB_ENV
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Set up GOPATH variable
+        run: echo "GOPATH=$(echo $HOME)" >> $GITHUB_ENV
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.3
+      
+      - name: Install aqueduct-ml
+        run: pip3 install aqueduct-ml
+
+      - name: Start the server
+        run: (aqueduct start > $SERVER_LOGS_FILE 2>&1 &)
+
+      - name: Wait for server
+        timeout-minutes: 1
+        run: while ! echo exit | nc localhost 8080; do sleep 1; done
+
+      # Grabs the pid of the process bound to port 8080 and kills it.
+      - name: Kill the server
+        run: kill -9 $(lsof -nP -iTCP -sTCP:LISTEN | grep 8080 | awk '{print $2}')
+
+      # install_local.py requires ~/.aqueduct/server scaffolding to exist.
+      - name: Update aqueduct with latest code
+        run: python3 scripts/install_local.py
+
+      - name: Start the server again
+        run: (aqueduct start > $SERVER_LOGS_FILE 2>&1 &)
+
+      - name: Install packages needed for testing
+        run: pip3 install nltk matplotlib pytest-xdist
+
+      - name: Wait for server again
+        timeout-minutes: 1
+        run: while ! echo exit | nc localhost 8080; do sleep 1; done
+
+      - name: Fetch the API key
+        run: echo "API_KEY=$(aqueduct apikey)" >> $GITHUB_ENV
+
+      - name: Run the Integration Tests
+        timeout-minutes: 10
+        working-directory: integration_tests/sdk
+        env:
+          SERVER_ADDRESS: localhost:8080
+          INTEGRATION: aqueduct_demo
+        run: pytest . -rP --publish -n ${{ matrix.concurrency }}
```

Comment:
are we running with `--complex-models` too or does that seem unnecessary.

Reply:
I could be convinced either way tbh

Reply:
we can run complex-models as well since these are run once a day and time is not an issue.